wolfSSL-JNI/src/chapter04.md

```diff
@@ -54,8 +54,8 @@ for (Provider prov:providers) {
 
 To install the wolfJSSE provider at the system/OS level, copy the `wolfssl.jar`
 and/or `wolfssl-jsse.jar` into the correct Java installation directory for your
-OS and verify the `libwolfssljni.so` or `libwolfssljni.dylib` shared library
-is on your library search path.
+OS and verify the `libwolfssljni.so`, `libwolfssljni.dylib`, or
```

Comment:
Like above, we can simplify this to remove reference of `libwolfssljni.dylib` here.